packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2594,14 +2594,14 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
     } else {
       assert(() {
         if (debugPrintMarkNeedsPaintStacks) {
-          debugPrintStack(label: 'markNeedsPaint() called for $this (root of render tree)');
```

Comment:
leave this in? It's still correct that this marks the root of the render tree

Reply:
done